<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_con</class>
 <widget class="QDialog" name="Dialog_con">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>157</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pass</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="le_pass">
     <property name="text">
      <string>pass</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Open enter</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="button_ok">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="le_log">
     <property name="text">
      <string>danfox</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_con</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_ok</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_con</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
